### PR TITLE
Multiple improvements

### PR DESCRIPTION
**Pull request by molant**
* Added appveyor to generate the binaries automatically on the cloud and publish them (unsigned) to the GitHub releases
* Create binaries for x86 and x64 for node 6 and 8
* Update dependencies to latest versions
* Add support for `node-pre-gyp` so people in the right platform don't need to install anything (will get binaries from GitHub releases)
* Updated documentation
* Updated version to `0.6.0-unsigned` so it's clear what's happening
* Fix issues installing the VC++ redistributable in x86 node

### CODE REVIEW

**Comment by molant** [comment]
I still need to update the `appveyor.yml` file to deploy only on tag or in a certain branch (still haven't decided) and also see if I need to update the `prebuild.cmd` (haven't been capable of building locally yet...).

This PR should take care of #18 (fixed in a previous PR), #34 (haven't seen any of those warnings in my tests), #47, and #56 that I'm aware.

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "edge-diagnostics-adapter",
-  "version": "0.5.0",
+  "version": "0.6.0-unsigned",
   "description": "Microsoft Edge Diagnostics Adapter",
-  "main": "out/src/edgeAdapter.js",
+  "main": "dist/edgeAdapter.js",
```
Comment (sarvaje) [inline package.json]: that file doesn't exist, it should it should be like it was before `out/src/edgeAdapter`

Comment (molant) [reply package.json]: I though I reverted this changed :/

File: src/adapterService/edgeAdapterService.ts [hunk 2]
```diff
@@ -1,442 +1,426 @@
...
+                edgeAdapter.serveChromeDevTools(chromeToolsPort);
+            }
+
+            if (url) {
+                edgeAdapter.openEdge(url);
+            }
+
+            this._httpServer = http.createServer((req, res) => this.onServerRequest(req, res));
+            this._webSocketServer = new WebSocketServer({ server: this._httpServer });
+            this._webSocketServer.on('connection', (client) => this.onWSSConnection(client));
```
Comment (sarvaje) [inline src/adapterService/edgeAdapterService.ts]: you need to add a second parameter to the event handler. Something like this:
`this._webSocketServer.on('connection', (client, message) => this.onWSSConnection(client, message));`

Comment (molant) [reply src/adapterService/edgeAdapterService.ts]: fixed and `protocol.json` updated. I'll need to manually update the release with the new files though

**Comment by sarvaje** [comment]
@molant right now there is an error if you launch `npm run build` because the env variable `VCTargetsPath` is not defined.

To make it works, you need to run first the `prebuild.cmd`. is it possible to add the value of that variable to the command `npm run build`?

**Comment by molant** [comment]
@sarvaje, done!

**Comment by molant** [comment]
Finished doing all the testing and things are working on a clean install of RS2. Merging!

**Pull request merged**